src/ListTabulator/index.ts

```diff
@@ -268,6 +268,80 @@ export default class ListTabulator {
     };
   }
 
+  /**
+   * Method that specified hot to merge two List blocks.
+   * Called by Editor.js by backspace at the beginning of the Block
+   *
+   * Content of the first item of the next List would be merged with deepest item in current list
+   * Other items of the next List would be appended to the current list without any changes in nesting levels
+   *
+   * @param {ListData} data - data of the second list to be merged with current
+   * @public
+   */
+  merge(data: ListData): void {
+    const blockItems = this.block.holder.querySelectorAll(`.${DefaultListCssClasses.item}`);
+
+    const deepestBlockItem = blockItems[blockItems.length - 1] as HTMLElement;
+    const deepestBlockItemContent = deepestBlockItem.querySelector(`.${DefaultListCssClasses.itemContent}`) as HTMLElement;
+
+    if (deepestBlockItem === null || deepestBlockItemContent === null) {
+      return;
+    }
+
+    focus(deepestBlockItemContent);
+
+    const restore = save();
+    /**
+     * Insert trailing html to the deepest block item content
+     */
+    deepestBlockItemContent.insertAdjacentHTML('beforeend', data.items[0].content);
+
+    restore();
+
+    if (this.listWrapper === undefined) {
+      return;
+    }
+
+    const firstLevelItems = this.getChildItems(this.listWrapper);
+
+    if (firstLevelItems === null) {
+      return;
+    }
+
+    /**
+     * Get last item of the first level of the list
+     */
+    const lastFirstLevelItem = firstLevelItems[firstLevelItems.length - 1];
+
+    /**
+     * Get child items wrapper of the last item
+     */
+    let lastFirstLevelItemChildWrapper = lastFirstLevelItem.querySelector(`.${DefaultListCssClasses.itemChildren}`);
+
+    /**
+     * Append child items of the first element
+     */
+    if (data.items[0].items.length !== 0) {
+      /**
+       * Render child wrapper of the last item if it does not exist
+       */
+      if (lastFirstLevelItemChildWrapper === null) {
+        lastFirstLevelItemChildWrapper = this.renderer.renderWrapper(false);
+      }
+
+      this.appendItems(data.items[0].items, lastFirstLevelItemChildWrapper);
+    }
+
+    /**
+     * Remove first item that is already merged
+     */
+    data.items.shift();
```

Comment:
just for beautifying:

Put it above the previous statement and use `firstItem` instead of `data.items[0]`

```ts
const firstItem = data.items.shift()
```